src/util.js

```diff
@@ -363,8 +363,9 @@ var ret = {
     markAsOriginatingFromRejection: markAsOriginatingFromRejection,
     classString: classString,
     copyDescriptors: copyDescriptors,
-    hasDevTools: typeof chrome !== "undefined" && chrome &&
-                 typeof chrome.loadTimes === "function",
+    hasDevTools: false,
+    // hasDevTools: typeof chrome !== "undefined" && chrome &&
+    //              typeof chrome.loadTimes === "function",
```

Comment:
Seems fine for now. So it sounds like this is a code path for resolving promises sync rather then resolving them async using the bluebird scheduler. Seems like that may be valuable for us for debugging errors in promise callstacks - it will allow us to get bluebird off the stack and perhaps keep more context from the previous Hadron stack. Also, seems like a valuable mechanism (along with batchSize) for helping flush out race conditions in tests.

Can you file a separate task (and tag here) to add a configuration param for enabling that code path from Hadron?